src/main/java/uk/gov/hmcts/ccd/data/user/UserService.java

```diff
@@ -58,6 +62,13 @@ public UserProfile getUserProfile() {
         return createUserProfile(idamProperties, userId, jurisdictionDefinitions);
     }
 
+    public List<CaseTypeDefinition> getUserCaseTypes() {
+        return Arrays.stream(getUserProfile().getJurisdictions())
+            .map(JurisdictionDisplayProperties::getCaseTypeDefinitions)
+            .flatMap(Collection::stream)
+            .collect(Collectors.toList());
+    }
```

Comment:
this is an interesting solution in general. But for ManagaCaseAssignment won't work I think. I've just been recently told that the only IDAM role the client has is `caseworker-caa` and she needs to see all the existing case types. So in this case this will return an empty list